src/lighteval/tasks/registry.py

```diff
@@ -97,14 +108,18 @@ def get_task_class(
         )
 
     def get_task_dict(
-        self, task_name_list: List[str], custom_tasks: Optional[Union[str, ModuleType]] = None
+        self,
+        task_name_list: List[str],
+        custom_tasks: Optional[Union[str, ModuleType]] = None,
```

Comment:
mhh so we keep both custom and extended ? Or custom here is meant to be community ?

Reply:
community and custom tasks can be launched here